src/order/rel_iso.lean

```diff
@@ -317,6 +319,12 @@ def of_strict_mono {α β} [linear_order α] [preorder β] (f : α → β)
 @[simp] lemma coe_of_strict_mono {α β} [linear_order α] [preorder β] {f : α → β}
   (h : strict_mono f) : ⇑(of_strict_mono f h) = f := rfl
 
+/-- Embedding of a subtype into the ambient type as an `order_embedding`. -/
+def subtype (p : α → Prop) : subtype p ↪o α :=
```

Comment:
Would it be more canonical to call this `subtype.order_embedding`?
Also, should there be a lemma showing that it's equal to `subtype.rel_embedding`? I think they should be defeq when this one is defined.

Reply:
I named it after `function.embedding.subtype`.